utils/build-script-impl

```diff
@@ -3448,7 +3448,7 @@ function build_and_test_installable_package() {
               call tar -c -z -f "${package_for_host}" "${TOOLCHAIN_PREFIX/#\/}"
         else
             # tar on OS X doesn't support --owner/--group.
```

Comment:
Nitpick: please change this comment too! It sounds like the real condition here is "BSD tar".